bucket/pentaho-aggregation-designer.json

```diff
@@ -0,0 +1,50 @@
+{
+    "homepage": "https://pentaho.sourceforge.io/",
+    "description": "The Pentaho Aggregation Designer simplifies the creation and deployment of aggregate tables that improve the performance of your Pentaho Analysis (Mondrian) OLAP cubes.",
+    "license": {
```

Comment:
It is valid SPDX identifier. Do not include URL.

```suggestion
    "license": "GPL-2.0-only|LGPL-2.0-only|MPL-1.1",
```
or 

```suggestion
    "license": "GPL-2.0-only",
```
![A](https://i.imgur.com/lCvDLbD.png)